spec.html

```diff
@@ -671,7 +671,7 @@ <h1>Grammar Notation</h1>
           `9`
       </emu-grammar>
       <p>If the phrase &ldquo;[empty]&rdquo; appears as the right-hand side of a production, it indicates that the production's right-hand side contains no terminals or nonterminals.</p>
-      <p>If the phrase &ldquo;[lookahead &notin; _set_]&rdquo; appears in the right-hand side of a production, it indicates that the production may not be used if the immediately following input token sequence is a member of the given _set_. The _set_ can be written as a comma separated list of one or two element terminal sequences enclosed in curly brackets. For convenience, the set can also be written as a nonterminal, in which case it represents the set of all terminals to which that nonterminal could expand. If the _set_ consists of a single terminal the phrase &ldquo;[lookahead &ne; _terminal_]&rdquo; may be used.</p>
+      <p>If the phrase &ldquo;[lookahead &notin; _set_]&rdquo; appears in the right-hand side of a production, it indicates that the production may not be used if the immediately following input token sequence is a member of the given _set_. The _set_ can be written as a comma separated list of one or two element terminal sequences enclosed in curly brackets. For convenience, the set can also be written as a nonterminal, in which case it represents the set of all terminals to which that nonterminal could expand. If the _set_ consists of a single terminal sequence the phrase &ldquo;[lookahead &ne; _terminal_ _sequence_]&rdquo; may be used.</p>
```

Comment:
I wonder if it should be _terminalSequence_ instead? Or is there a precedent for the space?

Reply:
It said ‘terminal sequences’ in the first paragraph already when describing set members. I don’t think it occurs anywhere else so it’s not much of a precedent. Camel cased, I’d kind of expect it to be a defined term — currently it’s just directly descriptive, meaning just ‘a sequence of terminals’.

Reply:
I just mean that `_this_` in ecmarkup denotes a [variable](https://bterlson.github.io/ecmarkup/#emd-overview). Sure, this one's ad hoc and meant to be self-explanatory, but it doesn't seem right to stick a space inside it.

Reply:
Makes sense & I agree. I’ll change it to a camelcase name if you think that’s the best course. Personally, I tend to think these aren’t vars in the usual sense and shouldn’t be marked up with `_underscores_` at all (see last paragraph of first message) but that seemed too opinionated of a change for me to make, so I figured I was stuck with the two words because that’s how it’s described in the preceding sentences.

Reply:
Sure. I'll defer to what others think, but assuming we keep the underscores, anything without a space is okay by me.

Reply:
Agree with @rkirsling. The easiest mental model seems to be that a single variable expands to a sequence of a terminals.